pkg/agent/openflow/client.go

```diff
@@ -1181,3 +1187,31 @@ func (c *client) SendUDPPacketOut(
 	packetOutObj := packetOutBuilder.Done()
 	return c.bridge.SendPacketOut(packetOutObj)
 }
+
+// AddMeterFlow adds meter qos flow
```

Comment:
qos -> QoS

Reply:
Thanks for the review, I will update it.